jazz_test-lambda/swagger/swagger.json

```diff
@@ -0,0 +1,419 @@
+            ".*\"NotFound\".*": {
+              "statusCode": "404",
+              "responseTemplates": {
+                "application/json": "$input.path('$.errorMessage')"
+              },
+              "responseParameters": {
+                "method.response.header.Access-Control-Allow-Headers": "'Content-Type,X-Amz-Date,Authorization,X-Api-Key'",
+                "method.response.header.Access-Control-Allow-Methods": "'*'",
+                "method.response.header.Access-Control-Allow-Origin": "'*'"
+              }
+            }
+          }
+        }
+      }
+    }
+  },
+  "securityDefinitions": {
+		"{envPrefix}-{domain}-cognito-authorizer": {
+		  "type": "apiKey",
+		  "name": "Authorization",
+		  "in": "header",
+		  "x-amazon-apigateway-authtype": "custom",
+		  "x-amazon-apigateway-authorizer": {
+			"authorizerCredentials": "{conf-role}",
+			"authorizerResultTtlInSeconds": 300,
+			"authorizerUri": "arn:aws:apigateway:{conf-region}:lambda:path/2015-03-31/functions/arn:aws:lambda:{conf-region}:{conf-accId}:function:{envPrefix}-{domain}-cognito-authorizer-{envmnt}/invocations",
+			"type": "token"
+		  }
+		}
+	},
+  "definitions": {
+    "200SuccessResponse": {
+      "type": "object",
+      "properties": {
+        "data": {
+          "$ref": "#/definitions/OutputObj"
+        },
+        "input": {
+          "$ref": "#/definitions/InputObj"
+        }
+      },
+      "title": "Success Response"
+    },
+    "400ErrorResponse": {
+      "type": "object",
+      "properties": {
+        "error_type": {
+          "type": "string",
+          "default": "Bad Request format"
+        },
+        "message": {
+          "type": "string"
+        }
+      }
+    },
+    "401ErrorResponse": {
+      "type": "object",
+      "properties": {
+        "errorType": {
+          "type": "string",
+          "default": "Unauthorized"
+        },
+        "message": {
+          "type": "string"
+        }
+      }
+    },
+    "403ErrorResponse": {
+      "type": "object",
+      "properties": {
+        "errorType": {
+          "type": "string",
+          "default": "Forbidden"
+        },
+        "message": {
+          "type": "string"
+        }
+      }
+    },
+    "404ErrorResponse": {
+      "type": "object",
+      "properties": {
+        "error_type": {
+          "type": "string",
+          "default": "Resource Not Found"
+        },
+        "message": {
+          "type": "string"
+        }
+      }
+    },
+    "500ErrorResponse": {
+      "type": "object",
+      "properties": {
+        "error_type": {
+          "type": "string",
+          "default": "Internal Server Error"
+        },
+        "message": {
+          "type": "string"
+        }
+      }
+    },
+    "InputObj": {
+      "type": "object",
+      "properties": {
+        "functionARN": {
+          "type": "string",
+          "description": "ARN For the lambda to be tested"
+        },
+        "inputJSON": {
+          "type": "string",

```

Comment:
Any reason why inputJSON is of type 'string'? I think this should of type (json) object.